Documentation drive (#1)
InterWasm/docs: docs/02-getting-started/05-interact-with-contract.md

# Uploading and Interacting

We have the binary ready. Now it is time to see some wasm action. You can use [Go CLI](#go-cli) or [Node Console](#node-console).

## Go CLI {#go-cli}

We generated a wasm binary executable in the previous chapter. Let's upload the code to the blockchain. Once that is complete, you can download the bytecode to verify it.

```shell

> # see how many contracts we have now

Reviewer: ```suggestion
# see how many code we have now
```

Reviewer: This is for listing the code not contract

### Instantiating the Contract {#instantiating-the-contract}

We can now create an instance of this wasm contract. Here the verifier will fund an escrow, that will allow fred to control payout and upon release, the funds go to bob.

```shell
# instantiate contract and verify

> # the --amount flag sets the amount the contract has control over

Reviewer: ```suggestion
# the --amount flag sets the amount of coin sent to the contracts address
```
